ga4gh/client.py

```diff
@@ -9,10 +9,23 @@
 import requests
 import posixpath
 import logging
+import webbrowser
 
 import ga4gh.protocol as protocol
 
 
+class OIDCSession(requests.Session):
+    def __init__(self, authzEndpoint, key=None):
+        super(OIDCSession, self).__init__()
+        self._authzEndpoint = authzEndpoint
+        if key:
+            self._key = key
+            self.headers['Authorization'] = "Bearer {0}".format(self._key)
+        else:
+            print("""No login key. Please go to {0} in a browser, and use the
```

Comment:
It would be good to abstract this... we want to avoid `print` calls in ga4gh code